### ISSUE TITLE
wrong place with the first point of lefteye landmarks

### ISSUE THREAD

**Issue by chky** [Joey0094/GazeML_torch#1]
Great job!
Thank you for your pytorch implementation of GazeML, However, I meet some problem when I load the pretrained file "models/v0.2/model-v0.2-(36, 60)-epoch-89-loss-0.7151.pth", I use the model to test on MPIIFaceGaze dataset, but I find the first point of lefteye landmarks always locates on the wrong place, some examples are listed here
![1](https://user-images.githubusercontent.com/33181792/222120264-4d87946a-283f-40a4-94be-d6d46ea10077.png)
![2](https://user-images.githubusercontent.com/33181792/222120273-cfcf337f-dce5-4907-9c14-881cfcb198d6.png)
![3](https://user-images.githubusercontent.com/33181792/222120277-b0802691-eb0e-44b2-95d7-324bc1fbdc45.png)
The first point of right landmarks locates on the right place, I'm quite puzzled

**Comment by woshigua94** [comment]
This project has been going on for a long time, and it’s hard for me to find out where the problem is. I suggest you check whether the pixel coordinates of the eye keypoints output by the neural network are correct before visualization. If they are not correct, then it’s a problem with the model. If they are correct, then it’s a problem with the subsequent visualization processing.

**Comment by chky** [comment]
Thanks for your reply. It is a problem of my visualization code exactly. The result is correct this time
![1](https://user-images.githubusercontent.com/33181792/222369121-557a6c3d-946b-42f5-888c-e77bd3762543.png)
